components/AppHead/AppHead.js

```diff
@@ -23,7 +23,7 @@ function AppHead({ Component, pageProps }) {
     if (_isNotBrowser) return null;
 
     // Only run Google Analytics and Facebook Pixels in production
-    if (!process.env.NODE_ENV === 'production') return null;
```

Comment:
this effectively is `if (false === 'production')` since `!process.env.NODE_ENV` will get eval'd to `false`